wiki/Beatmaps/en.md

```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _map_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
+In their most basic form, beatmaps have a song various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
```

Comment:
`In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.`